src/en/communities.md

```diff
@@ -0,0 +1,121 @@
+---
+outline: deep
+cSpell: locale en
```

Comment:
Macht es nicht sinn das über die vscode settings global einzustellen, anstatt bei jeder Datei ein `cSpell` drüber zu haben?

Reply:
Aktuell unterscheide ich hier zwischen deutscher und englischer Rechtschreibprüfung auf Dokumentebene. Mir ist bisher kein Weg bekannt, wie ich dies global einstellen kann, sodass die deutschen Artikel nach deutscher und die englischen Artikel nach englischer Rechtschreibung geprüft werden (z.B. auf Ordnerebene). Ein ständiges hin- und herwechseln in den VSCode Settings, wenn ich an beiden Sprachen parallel arbeite, habe ich nicht als effektiv empfunden. Aber ich bin für Vorschläge offen.